source/Mlos.Notebooks/OptimizerEvaluationTools.ipynb

```diff
@@ -19,7 +19,10 @@
     "# TODO\n",
     "\n",
     "1. Add ability to dump trace to file.\n",
```

Comment:
Remove the: 
* InspectingEvaluationReport
* OptimizingOptimizer notebooks

And make sure that OptimizerEvaluationTools contains a superset of that functionality.